charts/parca/templates/clusterrole.yaml

```diff
@@ -1,20 +1,23 @@
+{{- if and .Values.server.enabled .Values.server.rbac.create }}
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
-    {{- include "parca.labels" . | nindent 4 }}
   name: {{ include "parca.serviceAccountName" . }}
+  labels:
+    app: {{ include "parca.serviceAccountName" . }}
+{{- include "parca.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - ""
+- apiGroups: [""]
   resources:
+  - nodes
+  - nodes/metrics
```

Comment:
is this the permission that is checked to be allowed to scrape node profiling data? if not then I don't think we need this in Parca

Reply:
Only nodes is needed, i've removed the ingress section